### PR TITLE
feat: POC support multi `.md` files

### PR DESCRIPTION
**Pull request by cpreston321**
### 🔗 Linked issue

#21 

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme, or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [X] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Resolves: #21 

### 📝 Checklist

- [X] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by cpreston321** [comment]
Still work in progress but want to get feedback before I go any further.

File: docs/1.guide/2.config.md [hunk 1]
```diff
@@ -31,16 +32,23 @@ Working directory where paths are resolved from.
...
 
 > [!TIP]
 > You can use `--file` to override with CLI.
 
+### `dirs`
+
+- Type: `string[]`
+- Default: `[]`
+
+Directory paths to scan for markdown files (relative to `dir`).
```
Comment (uncenter) [inline docs/1.guide/2.config.md:50]:
```suggestion
Directory paths to scan for markdown files.
```

Comment (cpreston321) [reply docs/1.guide/2.config.md:50]: I want the user to know it's relative to the dir they set. I don't want them to set a full path so it won't be confusing

**Comment by pi0** [comment]
Thanks for spending time on this POC PR ❤️ 

I am working on a couple of reactors to unblock the direct `transform` possibility first.

It has already started to conflict with this branch changes, sorry :( I self-assigned to avoid this..

**Pull request closed without merging**